spec/latest/core/core.jsonld

```diff
@@ -505,6 +518,112 @@
       "domain": "hydra:IriTemplateMapping",
       "range": "xsd:string",
       "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:target",
+      "@type": "rdf:Property",
+      "label": "invocation target",
+      "comment": "Explicit target of the invocation of the operation.",
+      "domain": "hydra:Operation",
+      "rangeIncludes": ["hydra:IriTemplate", "hydra:Resource"],
+      "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:returnsHeader",
+      "@type": "rdf:Property",
+      "label": "returns header",
+      "comment": "Name of the header returned by the operation.",
+      "domain": "hydra:Operation",
+      "range": "xsd:string",
+      "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:expectsHeader",
+      "@type": "rdf:Property",
+      "label": "expects header",
+      "comment": "Specification of the header expected by the operation.",
+      "domain": "hydra:Operation",
+      "rangeIncludes": ["hydra:Header", "hydra:HeaderTemplate"],
+      "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:Template",
+      "@type": "rdfs:Class",
+      "label": "Template",
+      "comment": "Some abstract template.",
+      "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:Header",
+      "@type": "rdfs:DataType",
+      "label": "Header",
+      "comment": "Specification of the header, including name and value(s).",
+      "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:HeaderTemplate",
+      "@type": "rdfs:Class",
+      "subClassOf": "hydra:Template",
+      "label": "Header template",
+      "comment": "Specification of the header template with variables to be replaced with actual values.",
+      "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:headerName",
+      "@type": "rdf:Property",
+      "label": "header name",
+      "comment": "Name of the header.",
+      "domain": "hydra:HeaderTemplate",
+      "range": "xsd:string",
+      "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:MediaTypedResource",
+      "@type": "rdfs:Class",
+      "subClassOf": "hydra:Resource",
+      "label": "Resource with media type",
+      "comment": "Resource that has an RFC 6838 compliant media type specified.",
+      "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:mediaType",
+      "@type": "rdf:Property",
+      "label": "media type",
+      "comment": "Points to the RFC 6838 compliant media type specification.",
+      "domain": "hydra:MediaTypedResource",
+      "range": "hydra:MediaType",
+      "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:MediaType",
+      "@type": "rdfs:DataType",
+      "label": "Media type",
+      "comment": "An RFC 6838 compliant media type specification.",
+      "vs:term_status": "testing"
+    },

```

Comment:
The media type stuff is cool in principle but how does it relate to the other changes within this PR?

Reply:
I wanted to enable hydra to expect/return resources other than RDF. One of the approach of describing a resource would be to use the media type, i.e. for JPG or DOCX file that would be expected by the API